src/views/pages/LoginPage/index.jsx

```diff
@@ -48,22 +45,22 @@ const LoginPage = (props) => {
                 <img style={{ height: '300px' }} src='https://www.fitnesstogether.gr/wp-content/uploads/2021/06/Kick-Boxing-1-1024x529.jpg' alt='GTT logo' />
             </div>
             <div className='form-login'>
-                <div className='login'>
+                <form className='login' onSubmit={LoginRequest}>
                     <p>Already have an account?</p>
                     <FormInput
                         email={email}
                         password={password}
                         changeEmail={onChangeEmail}
                         changePassword={onChangePassword}
-                        error={error}
+                        error={isError}
                         isHide={isHide}
                         onClick={handleClickShowPassword}
                         isRegister={false}
                     />
-                    <button className='btn btn-login' onClick={LoginRequest}>Log In</button>
-                    {isLoggedIn && <Navigate to='/home' />}
+                    <button className='btn btn-login'>Log In</button>
```

Comment:
This button is not a submit then how can form's onSubmit is triggered when user press enter?

Reply:
I have fixed it.